### DISCUSSION TITLE
Both ugly and pretty shown

### DISCUSSION THREAD

**Discussion by brunoparga** [yoavbls/pretty-ts-errors#127] [Q&A] [answered]
Hi! Thank you for the extension. I am currently seeing _both_ the ugly error and the pretty one:
![image](https://github.com/user-attachments/assets/4ab53fe5-9ab6-42b2-8437-3167ff219a50)

Is this how it works for everyone? What can I do to ditch the ugly version and see only the pretty one? I'm running VSCodium on Manjaro Linux, FWIW. Everything should be up to date; in particular, the extension version I'm running is 0.5.3.

**Comment by yoavbls** [comment, marked as answer]
Hey, unfortunately, it is a known limitation: https://github.com/yoavbls/pretty-ts-errors/issues/3
But there is a CSS hack that allows you to hide the original error in the last comment in the issue
